data/korath/korath jobs.txt

```diff
@@ -243,3 +243,588 @@ mission "Kor Efreti to Wanderers: Farming"
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 5 10 0.8
+	illegal 50000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		has "event: know about Korath fish jobs"
+		not "Korath Humanika: Fish Trade (small): active"
+	on stopover
+		dialog "You drop of your <tons> of illicit fish, and receive <tons> of unknown cargo from a woman in a blue dress."
+	on complete
+		payment 25000 500
+		dialog "A woman in a blue dress thanks you for the delivery and pays you <payment>."
+		"reputation: Korath" ++
+
+mission "Korath Humanika: Fish Trade (medium)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 15 50 0.8
+	illegal 100000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		random < 20
+		has "event: know about Korath fish jobs"
+		not "Korath Humanika: Fish Trade (medium): active"
+		"cargo space" > 20
+	on stopover
+		dialog "You drop of your <tons> of illicit fish, and receive <tons> of unknown cargo from a woman in a blue dress."
+	on complete
+		payment 35000 400
+		dialog "A woman in a blue dress thanks you for the delivery and pays you <payment>."
+		"reputation: Korath" += 2
+
+mission "Korath Humanika: Fish Trade (bulk)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 40 300 0.8
+	illegal 150000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		random < 5
+		has "event: know about Korath fish jobs"
+		not "Korath Humanika: Fish Trade (bulk): active"
+		"cargo space" > 50
+	on stopover
+		dialog "You drop of your <tons> of illicit fish, and receive <tons> of unknown cargo from a woman in a blue dress."
+	on complete
+		payment 45000 350
+		dialog "A woman in a blue dress thanks you for the delivery and pays you <payment>."
+		"reputation: Korath" += 3
+
+mission "Korath: Retrieve livestock [0]"
+	name "Livestock from <stopovers>"
+	job
+	repeat
+	description "Take <tons> of trade goods to <stopovers> to trade for <tons> of unique livestock and return to <planet> for <payment>."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "livestock trade" 1 4 0.8
+	source "La'ab Bamen"
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes farming
+	to offer
+		random < 70
```

Comment:
![image](https://user-images.githubusercontent.com/4471575/221462335-58216dae-dee2-48ea-afc4-d6c039f2e53f.png)

This is a pretty high random chance. While they're all missions that require you to visit a wide variety of locations, the odds should probably be toned back so you don't have an entire ring full of missions every time. Maybe 30-40%?

Reply:
That was leftover from debugging. I will fix it.

Reply:
I experimented, and for a small ship, 40% works well.